packages/ndla-ui/src/CreatedBy/CreatedBy.tsx

```diff
@@ -29,26 +36,22 @@ const StyledSafeLink = styled(SafeLink)<LinkProps>`
 `;
 
 type Props = {
-  contentUrl?: string;
+  name: string;
+  description: string;
+  url?: string;
 };
 
-const CreatedBy = ({ contentUrl, t }: Props & tType) => (
+const CreatedBy = ({ name, description, url }: Props) => (
```

Comment:
Vi må huske at dette er breaking for ndla-frontend så vi bør ha ny minor-versjon av ndla-ui.

Reply:
Godt poeng